en/SUMMARY.md

```diff
@@ -61,6 +61,9 @@
 * Middleware and Architecture
   * [uORB Messaging](middleware/uorb.md)
   * [MAVLink Messaging](middleware/mavlink.md)
+  * [DDS/ROS2 interface](middleware/micrortps.md)
+    * [MicroRTPS HelloWorld Example](middleware/micrortps_hello_world.md)
```

Comment:
For the sidebar (only) we could arguably remove "MicroRTPS" because you get the context from the parent item